buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -906,6 +915,10 @@ class BuildPlugin implements Plugin<Project> {
             File heapdumpDir = new File(project.buildDir, 'heapdump')
 
             project.tasks.withType(Test) { Test test ->
+                RepositoryHandler repos = project.repositories
+                if (project.ext.inFipsJvm) {
+                    project.dependencies.add('testRuntimeOnly', "org.bouncycastle:bc-fips:1.0.1:jar")
```

Comment:
I don't think we carry this classpath tot he clusters running in the rest tests, so these will probably not be running with fips enabled. We should have a smoke test to verify that fips is really enabled everywhere we think it is when we build.

Reply:
> Maybe I missed this but it seems this won't for rest tests as the tests themselves will run on fips jvm but the cluster will not.

Good catch, thanks

How about the integTest clusters? Those are covered by the above, correct ?

Reply:
yes internal clusters run in the same jvm as tests so would are covered.